src/models/transactions/paymentTransaction.ts

```diff
@@ -19,3 +19,80 @@ export interface PaymentTransaction extends BaseTransaction {
     DeliverMin?: Amount
     Flags?: PaymentTransactionFlags
 }
+
+/**
+ * @param {PaymentTransaction} tx A Payment Transaction.
+ * @returns {void}
+ * @throws {ValidationError} When the PaymentTransaction is malformed.
+ */
+export function verifyPaymentTransaction(tx: PaymentTransaction): void {
+    verifyBaseTransaction(tx)
+    
+    if (tx.Amount === undefined)
+        throw new ValidationError('PaymentTransaction: missing field Amount')
+    
+    if (typeof tx.Amount !== 'string' && !isIssuedCurrency(tx.Amount))
```

Comment:
This line can be replaced by just `isAmount(tx.Amount)`. (importing it from common!)